performance/default/index.js

```diff
@@ -1,5 +1,5 @@
-import { render, html } from '../../x-template.js';
-import CommonTest, { HtmlLiteralInterface } from './common.js';
+import { render, html } from '@netflix/x-element/x-template.js';
```

Comment:
I also added an import map to make these demo / performance imports “look more familiar”. I.e., from the demo’s perspective, we are now treating the dependency as “external” (even though it is indeed mapping to a local file within the import map).